pkg/reconciler/common/images_test.go

```diff
@@ -436,99 +435,3 @@ func TestImagePullSecrets(t *testing.T) {
 		})
 	}
 }
-
-func makeDaemonSet(name string, podSpec corev1.PodSpec) *appsv1.DaemonSet {
-	return &appsv1.DaemonSet{
-		TypeMeta: metav1.TypeMeta{
-			Kind: "DaemonSet",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name: name,
-		},
-		Spec: appsv1.DaemonSetSpec{
-			Template: corev1.PodTemplateSpec{
-				Spec: podSpec,
-			},
-		},
-	}
-}
```

Comment:
this was was already in the `util.go` file.

Moving the rest, for better reusage.